tests/test_datasets/test_utils.py

```diff
@@ -197,18 +197,21 @@ def test_get_target_with_quantiles(segments, columns, answer):
     assert targets_names == answer
 
 
-@pytest.mark.parametrize("target_level,answer_name", (("market", "market_level_df"), ("total", "total_level_df")))
-def test_get_level_dataframe(product_level_simple_hierarchical_ts, target_level, answer_name, request):
-    ts = product_level_simple_hierarchical_ts
-    answer = request.getfixturevalue(answer_name)
+@pytest.mark.parametrize(
+    "target_level, answer_name",
+    (("market", "market_level_constant_forecast_w_quantiles"), ("total", "total_level_constant_forecast_w_quantiles")),
+)
+def test_get_level_dataframe(product_level_constant_forecast_w_quantiles, target_level, answer_name, request):
+    ts = product_level_constant_forecast_w_quantiles
+    answer = request.getfixturevalue(answer_name).to_pandas()
     answer.index.freq = "D"
```

Comment:
Why is it like this? Where do we lose `freq`?

Reply:
It is for `pd.testing`

Reply:
Will check the entire fixture

Reply:
Fixed